org.otfried.Ipe.yaml

```diff
@@ -0,0 +1,125 @@
+app-id: org.otfried.Ipe
+
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+sdk: org.kde.Sdk
+
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --filesystem=xdg-documents:rw
+  - --env=IPELATEXPATH=/app/texlive/bin/x86_64-linux
```

Comment:
What will happen on aarch64?

Reply:
Good question. Can I have architecture-dependent `finish-args`?

Reply:
no. 

Either you add both (if it work like PATH), or a wrapper script that set the environment.